config/routes.js

```diff
@@ -56,7 +56,7 @@ module.exports.routes = {
   'post /api/v1/pokemon/multi': 'PokemonController.uploadMultipleFiles',
 
   'get /api/v1/pokemon/:id': 'PokemonController.get',
-  'get /api/v1/pokemon/:id/pk6': 'PokemonController.download',
+  'get /api/v1/pokemon/:id/pk': 'PokemonController.download',
```

Comment:
`raw` tells you that you're getting a raw file. `pk` just doesn't tell you anything. If we're breaking backwards compatibility anyway, it doesn't really matter whether it's consistent with what we had before.

(That's my whole spiel, I'm not going to contest this further if you still disagree.)